include/crm/common/ordering.h

```diff
@@ -22,7 +22,10 @@ extern "C" {
 
 //! Ordering constraint options
 enum pe_ordering {
-    pe_order_none                  = 0x0,       /* deleted */
+    //! Ordering disabled (compare with equality rather than bit set)
+    pcmk_no_ordering                    = 0U,
```

Comment:
I think none makes sense. I had considered it but nothing is really perfect due to how the ordering code works. I didn't want pcmk_ord_disabled because pcmk_ord_disabled|other_flag would be incorrect (though no_ordering isn't much better on that count -- I think none is most understandable in that construction). I decided not to use the prefix because it isn't a bit, and has to be compared with equality rather than is_set(), and I thought that would help. But that's true of all our bit enums, the ordering code just happens to check it more. So none is probably best for consistency with other enums.